substrate/frame/balances/src/lib.rs

```diff
@@ -1248,5 +1278,43 @@ pub mod pallet {
 			});
 			Ok(actual)
 		}
+
+		/// Generate dev account from derivation string.
+		#[cfg(feature = "runtime-benchmarks")]
+		pub fn derive_dev_account(num_accounts: u32, balance: T::Balance, derivation: &String) {
+			// Ensure that the number of accounts is not zero
+			assert!(num_accounts > 0, "num_accounts must be greater than zero");
+
+			assert!(
+				balance >= <T as Config<I>>::ExistentialDeposit::get(),
+				"the balance of any account should always be at least the existential deposit.",
+			);
+
+			assert!(
+				derivation.contains("{}"),
+				"Invalid derivation string"
```

Comment:
```suggestion
				"Invalid derivation, expected `{}` as part of the derivation"
```

Or something like this, to make it more obvious on why the string is invalid.